node/src/prover/mod.rs

```diff
@@ -110,6 +110,7 @@ impl<N: Network, C: ConsensusStorage<N>> Prover<N, C> {
             trusted_peers,
             Self::MAXIMUM_NUMBER_OF_PEERS as u16,
             matches!(storage_mode, StorageMode::Development(_)),
+            false,
```

Comment:
> https://github.com/AleoHQ/snarkOS/pull/3163/files#r1525287483

Should this be `true` as well according to your comment above?

Reply:
Yes